src/components/SearchBar.js

```diff
@@ -66,11 +84,19 @@ type SearchBarDefaultProps = {
   autoCompleteUri: string | null;
   route: string | null;
   baseUri: string;
+  allowSavedSearch: boolean;
```

Comment:
New props should be optional for backwards compatibility.
`allowSavedSearch?: boolean;`